### PR TITLE
Feature/new user controller

### PR DESCRIPTION
**Pull request by pawelbiesiada**

### CODE REVIEW

File: WebApp/WebApp/Logger.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace WebApp
+{
+    public class Logger
+    {
+        public void Debug()
+        {
+
```
Comment (agnieszkaanna) [inline WebApp/WebApp/Logger.cs:7]: Method not implemented

**Review by agnieszkaanna — CHANGES_REQUESTED** [review]
Do poprawy

File: WebApp/WebApp/Logger.cs (hunk 1 above)
Comment (pawelbiesiada) [reply WebApp/WebApp/Logger.cs]: Fixed

**Review by agnieszkaanna — APPROVED** [review]
Zaakceptowane

**Pull request merged**